### PR TITLE
product image add

### PR DESCRIPTION
**Pull request by Zorurunn**
add productt image ynzlaw

### CODE REVIEW

**Review by bekk1029 — APPROVED** [review]

File: .history/api/src/controllers/order.controller_20240325165456.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        return res.status(400).json({ message: "product id not found" });
+      }
+
+      const productStockQty = productStock.qty;
+
+      if (productStockQty < num || num < 0) {
+        return res.json({ message: "order exceeded stock" });
+      }
+
+      // const updateLeftProductQty = await ProductModel.findOneAndUpdate(
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165456.ts:59]: too this

File: .history/api/src/controllers/order.controller_20240325165456.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    }
+    console.log("duusaw");
+
+    return res.json({ message: "Successfully order added" });
+  } catch (error) {
+    res.json(error);
+  }
+};
+
+// BILKWRITE MINUS STOCK
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165456.ts:77]: too this

File: .history/api/src/controllers/order.controller_20240325165554.ts [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  quantity: number;
+  thumbnailUrl: string;
+};
+
+// CREATE ORDER
+export const createOrder: RequestHandler = async (req, res) => {
+  const { userId, deliveryDetails, orderedProducts, deliveryStatus } = req.body;
+
+  try {
+    // CHECK STOCK
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165554.ts:22]: too this

File: .history/api/src/controllers/order.controller_20240325165554.ts [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        return res.status(400).json({ message: "product id not found" });
+      }
+
+      const productStockQty = productStock.qty;
+
+      if (productStockQty < num || num <= 0) {
+        return res.json({ message: "order exceeded stock" });
+      }
+
+      // const updateLeftProductQty = await ProductModel.findOneAndUpdate(
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165554.ts:59]: too this

File: .history/api/src/controllers/order.controller_20240325165815.ts [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
+// );
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165815.ts:128]: ene file deer bga buh comment iig arilga

File: .history/api/src/controllers/order.controller_20240325165843.ts [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
+// );
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165843.ts:120]: end bga buh comment bolon console.log - g arilga

File: .history/api/src/controllers/order.controller_20240325165848.ts [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+//   );
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165848.ts:105]: too this

File: .history/api/src/controllers/order.controller_20240325165852.ts [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+//   );
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165852.ts:102]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325165909.ts [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+    }
+  );
+}
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165909.ts:101]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325165936.ts [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+//   );
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165936.ts:103]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325165952.ts [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
+// );
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325165952.ts:106]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325170030.ts [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
+// );
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325170030.ts:111]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325170033.ts [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
+// );
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325170033.ts:111]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325170034.ts [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
+// );
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325170034.ts:111]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325170036.ts [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+//   );
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325170036.ts:110]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325170041.ts [hunk 16]
```diff
@@ -0,0 +1,111 @@
...
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
+// );
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325170041.ts:111]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325170049.ts [hunk 17]
```diff
@@ -0,0 +1,111 @@
...
+//   );
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325170049.ts:110]: too this. (comment and console.log)

File: .history/api/src/controllers/order.controller_20240325170107.ts [hunk 18]
```diff
@@ -0,0 +1,111 @@
...
+// }
+
+// Too shirheg < stovk
+// bulkwrite;
+// await ProductModel.updateOne(
+//   { _id: "" },
+//   {
+//     $inc: { stock: -5 },
+//   }
+// );
```
Comment (BilguunTu) [inline .history/api/src/controllers/order.controller_20240325170107.ts:111]: too this. (comment and console.log)

File: api/src/controllers/order.controller.ts [hunk 19]
```diff
@@ -16,35 +16,38 @@ type orderedProductsType = {
...
       }
     }
-    // Too shirheg < stovk
-    // bulkwrite;
-    // await ProductModel.updateOne(
-    //   { _id: "" },
-    //   {
-    //     $inc: { stock: -5 },
-    //   }
-    // );
```
Comment (BilguunTu) [inline api/src/controllers/order.controller.ts:47]: comment clear

**Review by BilguunTu — COMMENTED** [review]
history arilgaad comment bolon console.log uudiig arigaad ywuulaarai

**Review by ulaaka211 — APPROVED** [review]

**Pull request merged**